src/styles/content.scss

```diff
@@ -57,9 +60,177 @@ table.os-raise-standardtable {
   }
 }
 
+%os-raise-basictable {
+  /* border-collapse: separate paired with border-spacing: 0 allows for a border-radius
+     to be applied to the entire table (border-collapse: collapse will ignore any border
+     radius applied to the table)
+  */
+  border: 2px solid $raise-table-color;
+  border-collapse: separate;
+  border-spacing: 0;
+  border-radius: 5px;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
+    font-weight: 600;
+    text-align: left;
+  }
+
+  thead {
+    background-color: $raise-table-color;
+  }
+
+  th,
+  td {
+    text-align: center;
+    padding: 16px;
+  }
+
+  th {
+    font-size: 1.2rem;
+    font-weight: 700;
+    color: #fff;
+  }
+
+  td {
+    font-size: 1.067rem;
+    color: $raise-light-black;
+    background-color: #fff;
+    border-left: 1px solid $raise-table-color;
+    border-top: 1px solid $raise-table-color;
+  }
+}
+
+table.os-raise-doubleheadertable {
+  table-layout: fixed;
+  width: 100%;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
+    font-weight: 600;
+    text-align: left;
+  }
+
+  th,
+  td {
+    border: 1px solid $raise-light-gray;
+    text-align: center;
+    padding: 16px;
+  }
+
+  th {
+    color: #000;
+    font-size: 1.2rem;
+    background-color: #84ada8;
+    font-weight: 700;
+  }
+
+  thead tr th:first-child {
+    border: none;
+    background-color: #fff;
+  }
+
+  td {
+    color: $raise-light-black;
+    font-size: 1.067rem;
+    background-color: #fff;
+  }
+}
+
+table.os-raise-textheavytable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 100%;
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  th,
+  td {
+    text-align: left;
+  }
+
+  td:first-child {
+    border-left: none;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+}
+
+table.os-raise-midsizetable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 66%;
+  margin: auto;
+
+  td:first-child {
+    border-left: none;
+  }
+}
+
+table.os-raise-skinnytable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 240px;
+
+  td:first-child {
+    border-left: none;
+  }
+}
+
+.os-raise-skinnytablemedia {
+  margin: auto;
+}
+
+table.os-raise-wideadjustedtable {
+  @extend %os-raise-basictable;
+
+  width: 100%;
+
+  td:first-child {
+    border-left: none;
+  }
+}
+
+table.os-raise-wideequaltable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 100%;
+
+  td:first-child {
+    border-left: none;
+  }
+}
+
+table.os-raise-horizontaltable {
+  @extend %os-raise-basictable;
+
+  width: 100%;
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  th {
+    background-color: $raise-table-color;
+    text-align: left;
+  }
+
+  tr:first-child td {
+    border-top: none;
+  }
+}
+
 .os-raise-d-flex {
   display: flex;
   flex-wrap: wrap;
+  row-gap: 5rem;
```

Comment:
I'm curious as to the value of `5rem` here since that'll convert to `80px` which ends up looking like:

![image](https://user-images.githubusercontent.com/839025/236949388-8adafe8d-0278-4293-992c-b85183199710.png)

This might be worth a quick check with UX to see if it looks okay :man_shrugging: .

Reply:
I have no preference in regards to how much space there is between rows, so I just put a value in that I thought created enough space between the tables.

@madisonschear - Whenever the browser window is not large enough to allow the tables to be on the same row they will drop down a row. Previously we didn't have any space between the tables when they dropped down a row so I incorporated that concept here. Could you let me know if 5rem works for you or if you'd like a different value?

Reply:
This works for me.

Reply:
Can we set the spacing to shrink based on the browser window? So that it shrinks to a minimum of spacing before dropping down to a new row?

Reply:
Are you referring to the spacing as seen in the screenshot? If so, then that spacing will already shrink until the tables are just about touching and then a table drops to a new row.

![verticalspacing](https://github.com/openstax/k12-apps-raise/assets/37000585/16c2198f-5959-4f77-9be9-58eb3b13af2c)